src/Database/ExpressionInterface.php

```diff
@@ -35,7 +35,7 @@ public function sql(ValueBinder $generator): string;
      * being iterated.
      *
      * @param callable $visitor The callable to apply to all nodes.
-     * @return $this
+     * @return $this|void
```

Comment:
This cannot be. Didnt I remove this as the void is only inside the callable etc?

Reply:
`traverse()` method of all expression classes which implement this interface return void. Only `Query::traverse()` returns `$this`.

Reply:
Couldnt all return this then? for a clean api?

Reply:
Fine by me if @lorenzo / @markstory agree.

Reply:
Going from void -> this is safe.